helpdesk_fieldservice/models/helpdesk_ticket.py

```diff
@@ -78,3 +81,14 @@ def action_create_order(self):
         res = self.env.ref('fieldservice.fsm_order_form', False)
         result['views'] = [(res and res.id or False, 'form')]
         return result
+
+    @api.depends('fsm_order_ids', 'stage_id', 'fsm_order_ids.stage_id')
+    def _compute_all_closed(self):
+        for ticket in self:
+            ticket.all_orders_closed = True
+            if ticket.fsm_order_ids:
+                for order in ticket.fsm_order_ids:
+                    if order.stage_id.name not in  ['Closed', 'Cancelled']:
+                        ticket.all_orders_closed = False
+            else:
+                ticket.all_orders_closed = False
```

Comment:
@osi-scampbell Do we really need the `else` statement here? If the ticket does not have FSM orders, then they are all closed and all_orders_closed = True

Reply:
@max3903 That is true, however the purpose of the filter was because employees were creating tickets, adding orders, closing the orders, but forgetting to close the tickets. Tickets that did not have orders were being handled fine to my knowledge. The purpose of the else statement is to separate the two cases.